docker/docker-compose.yml

```diff
@@ -3,98 +3,69 @@ services:
 
   # bootnode
   damian:
-    image: aleph-node:latest
+    image: aleph-node:${VERSION}
     container_name: damian
     privileged: true
     network_mode: host
-    entrypoint: ["/bin/sh","-c"]
-    command:
-      - |
-         ./aleph-node purge-chain --chain /tmp/chainspec.json --base-path /tmp/Damian -y
-         ./aleph-node --validator \
-                      --name Damian \
-                      --base-path /tmp/Damian \
-                      --node-key-file /tmp/Damian/libp2p_secret \
-                      --rpc-port 9933 --ws-port 9943 --port 30333 \
-                      --rpc-cors all --rpc-methods Safe \
-                      --execution Native \
-                      --no-mdns --no-prometheus --no-telemetry \
-                      --chain /tmp/chainspec.json \
-                      --bootnodes '/ip4/127.0.0.1/tcp/30333/p2p/12D3KooWAbWDCsZiA6EXrXh33E7hBYUFK53b5CSajoPsYL5n2Zdm'
+    ports:
```

Comment:
this is incompatible with host networking, might be the easiest to just remove the binding (see general comment).

Reply:
Indeed. Haven't noticed that ports were misconfigured here ( partially host network used, partially ports ).

To be honest, I would prefer port binding, but as it is easy to support both of the modes - let's use one that suits dev needs more.